spec/support/schemas/intent_to_file.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this used separately from the `intent_to_files` json below? are both necessary?

Reply:
`intent_to_files` will return an array of `intent_to_file` base objects. `intent_to_file` will return just a single object. Hence, the separation.

Reply:
yikes, that naming is a bit rough, but I see the point of it. carry on!